### ISSUE TITLE
Determine Parameterization for Running the LTI Build

### ISSUE THREAD

**Issue by anusreelakshmi934** [OpenLiberty/liberty-tools-intellij#783]
Determine how to run the LTI build by using 

- Lsp4ij plugin from marketplace 
- Lsp4ij plugin build locally

**Comment by anusreelakshmi934** [comment]
Planning on running a CronJob which checkouts the main branch of lsp4ij repo and runs the LTI tests.  Added an env variable  USE_LOCAL_PLUGIN which is set to true or false in the workflow file. In cronJob.yaml file it is set to true, so that it builds plugin locally while in build.yaml file it is set to false, so it builds plugin from the marketplace. The tasks in src/test/resources/ci/scripts/run.sh is also modified accordingly. 
Attaching the reference branch - https://github.com/anusreelakshmi934/liberty-tools-intellij/tree/issue%23783

**Comment by anusreelakshmi934** [comment]
Modified to use a single workflow file. Added an input parameter ( useLocalPlugin ) for workflow_dispatch to decide whether to use the local lsp4ij build. Used if conditions to include steps for checking out and building lsp4ij only if USE_LOCAL_PLUGIN is true. Attaching the branch - https://github.com/anusreelakshmi934/liberty-tools-intellij/tree/parameterizeIssue%23783

**Comment by anusreelakshmi934** [comment]
I also tested if the workflow fails when there are errors in the checked-out lsp4ij repository that cause our UI tests to fail. To do this, I purposefully commented out some code in my fork of the lsp4ij repo. This caused the UI tests to fail, confirming that the workflow is running tests from the checked-out repository specified in the workflow file.

For reference, here is the branch where I made the changes: [testingFailure](https://github.com/anusreelakshmi934/liberty-tools-intellij/tree/testingFailure). I ran two builds: one using the lsp4ij plugin [built locally](https://github.com/anusreelakshmi934/liberty-tools-intellij/actions/runs/9460324906/job/26058923213) ( USE_LOCAL_PLUGIN: true ) and the other with the lsp4ij plugin from the [marketplace](https://github.com/anusreelakshmi934/liberty-tools-intellij/actions/runs/9460322832/job/26061088923) ( USE_LOCAL_PLUGIN: false ). In the case of the locally built plugin, the tests failed, confirming the workflow is correctly running tests from the specified repo in the workflow file.
